index.html

```diff
@@ -12,11 +12,12 @@
 <body>
   
   <header>
+    <button class="nav-button1"> 
+      <a href="#about"></a>
+      <span class="logo">Joy</span>
```

Comment:
- Hey, great job, however, you need to wrap the logo inside the anchor tag to make it a link. please, fix this issue as suggested by the previous reviewer.

Reply:
Thank you. Kindly help me know: Does this mean I will also remove the <button> tag to be left with only the <a> tag?